<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>283</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Step 9 Make Report</string>
  </property>
  <widget class="QPushButton" name="btn_Process">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/btn_ok.png</normaloff>images/btn_ok.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Exit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/btn_Exit.png</normaloff>images/btn_Exit.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Summary File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Browse">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Browse</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/btn_File_Browser.png</normaloff>images/btn_File_Browser.png</iconset>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_SummaryFile">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>30</y>
     <width>505</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unit Price File:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_PriceFile">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>73</y>
     <width>505</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Browse_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Browse</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/btn_File_Browser.png</normaloff>images/btn_File_Browser.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>107</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Country:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_Country">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>116</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_Period">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>114</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>104</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Period:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
